### PR TITLE
My best work has been done

### PR DESCRIPTION
**Pull request by Victoria-Fire**

### CODE REVIEW

File: Hero/Super_Hero/Batman.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+```
+This new cjde for my test repository
+```
+
+My best friend is Supermsn
+
+```
+Save in memory
+```
```
Comment (Superman-777) [inline Hero/Super_Hero/Batman.md:11]: What do you want?

1. What
2. do
3. you
4. want

- 1
- 2
- 3

- [ ] ok

**Review by Superman-777 — CHANGES_REQUESTED** [review]
See notes

File: Hero/Super_Hero/Batman.md (hunk 1 above)
Comment (Victoria-Fire) [reply Hero/Super_Hero/Batman.md:11]: Good

File: Hero/Super_Hero/Batman.md [hunk 2]
```diff
@@ -4,8 +4,14 @@ I am dark man.
...
+My best friend is Superman
 
 ```
-Save in memory
+Save in your heart
+```
+
+Today is beautiful day!!!
+
+```
```
Comment (Superman-777) [inline Hero/Super_Hero/Batman.md:15]: Deleted it

**Review by Superman-777 — CHANGES_REQUESTED** [review]
See notes

**Review by Superman-777 — APPROVED** [review]
Lgfm

File: Hero/Super_Hero/Batman.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+I am dark man.
+
+```
+This new cjde for my test repository
```
Comment (Superman-777) [inline Hero/Super_Hero/Batman.md:4]: what new?

**Review by Superman-777 — CHANGES_REQUESTED** [review]
see notes

**Review by Superman-777 — APPROVED** [review]
Lgfm

**Comment by Victoria-Fire** [comment]
Tester_2 deleted

**Pull request merged**